package/MDAnalysis/core/groups.py

```diff
@@ -2222,8 +2222,8 @@ def __getattr__(self, attr):
             # eg:
             # if attr in _ATTR_ERRORS:
             # raise NDE(_ATTR_ERRORS[attr])
-            raise NoDataError("AtomGroup.{} not available; this requires Bonds"
-                              "".format(attr))
+            raise NoDataError("AtomGroup.{} not available; The attribute/method you are trying to access is not available because your topology doesn't contain information about {attr}"
+                              .format(attr))
```

Comment:
You're using a named placeholder in your format string but only passing it a value rather than a key/value pair, so this will presumably raise a `KeyError`

Reply:
Hi @fiona-naughton. Oops! Thank you for pointing that out.